haros/haros.py

```diff
@@ -317,10 +319,14 @@ def main(argv = None, source_runner = False):
                             level = logging.DEBUG)
     else:
         logging.basicConfig(level = logging.WARNING)
+
     try:
+        if args.dir:
+            os.chdir(args.dir)
```

Comment:
I don't want to add to your workload, but might be nice to restore the `cwd` upon completion of the command?